common.proto

```diff
@@ -111,6 +112,13 @@ enum DataType {
   JSON = 15;
 }
 
+message DataTypeParams {
```

Comment:
This is not in production yet, we shouldn't add it to `common.proto`. It should go into `common_v2.proto`